File: week1/Cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars = 0;
    int nr = 0;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (!(dollars > 0));
    int cents = round(dollars * 100);

    while (cents > 0)
    {
        if (cents >= 25)
        {
            cents -= 25;
            nr++;
        }
        else if (cents >= 10)
        {
            cents -= 10;
            nr++;
        }
        else if (cents >= 5)
        {
            cents -= 5;
            nr++;
        }
        else
        {
            cents -= 1;
            nr++;
        }
    }
    printf("%i\n", nr);
}

File: pset6/Cash/cash.py
from cs50 import get_float

dollars = 0
nr = 0

while (dollars <= 0):
    dollars = get_float("Change owed: ")

cents = round(dollars * 100, 0)

while (cents > 0):
    if (cents >= 25):
        cents -= 25
        nr += 1
    elif (cents >= 10):
        cents -= 10
        nr += 1
    elif (cents >= 5):
        cents -= 5
        nr += 1
    else:
        cents -= 1;
        nr += 1

print(nr)
